src/utils/saml.ts

```diff
@@ -81,72 +134,70 @@ export const getPreValidateResponse = (
     "Response"
   );
 
+  const maybeIdpIssuer = getSamlIssuer(doc);
+
   const hasStrictValidation = pipe(
     O.fromNullable(strictValidationOptions),
     O.chain(_ =>
```

Comment:
Please give `_` a meaningful name, or @michaeldisaro gets mad

Reply:
what do you think about `validationOptions` for that parameter?